ghproxy/ghcache/ghcache.go

```diff
@@ -76,6 +92,47 @@ func cacheResponseMode(headers http.Header) string {
 	return ModeMiss
 }
 
+type request struct {
+	request *http.Request
+	output  chan<- response
+}
+
+type response struct {
+	response *http.Response
+	err      error
+}
+
+func newThrottlingTransport(maxConcurrency int, delegate http.RoundTripper) http.RoundTripper {
+	input := make(chan request, 100)
```

Comment:
100?

Reply:
Yeah, should we make that configurable or just set a large enough value? Callers blocking isn't really an issue since they'd be waiting later anyway...